api/server.js

```diff
@@ -22,6 +21,7 @@ import { certificationSessionRoutes } from './src/certification/session-manageme
 import { devcompRoutes } from './src/devcomp/routes.js';
 import { evaluationRoutes } from './src/evaluation/routes.js';
 import { identityAccessManagementRoutes } from './src/identity-access-management/application/routes.js';
+import * as serverAuthentication from './src/identity-access-management/infrastructure/serverAuthentication.js';
```

Comment:
Les fichiers sont plutôt nommés en Kebab-case (sauf pour les modèles en Pascal case)
```suggestion
import * as serverAuthentication from './src/identity-access-management/infrastructure/server-authentication.js';
```

Reply:
Bien reçu. Je change !

Reply:
Voilà le fichier a été renommé en `server-authentication.js`.